src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -1,4 +1,4 @@
-﻿﻿﻿using System;
+using System;
```

Comment:
There were some really weird byte-order marks that VS removed:

![image](https://user-images.githubusercontent.com/840039/60733842-a7356480-9f13-11e9-94fe-593ff9beb6df.png)

Apparently three of them?

Reply:
The [BOM in UTF-8 is 3 bytes](https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8): 0xEF, 0xBB, 0xBF.  This is thus "expected."

What *isn't* "expected" is the removal of the BOM in the first place.  Why was the BOM removed?

Reply:
@jonpryor our `.editorconfig` is set to remove BOMs: https://github.com/xamarin/xamarin-android/blob/0b79ab6653a4364f837d513453567485c535c611/.editorconfig#L9

I can change this to `charset = utf-8-bom`, and it will add BOMs by default.

However, I think it is pretty useful to remove the BOMs for proguard and java files.

Reply:
Can that config file not set that setting per file type?

Reply:
The `utf-8-bom` setting does this:

![image](https://user-images.githubusercontent.com/840039/60820646-7cdbe500-a167-11e9-92e7-24e986925a9a.png)

Which still appears to be different than what was in this file. But it matches other files?

We can change the BOM setting on a per-extension basis, if that is what we want to do.